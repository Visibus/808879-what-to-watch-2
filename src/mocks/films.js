const films = [
  {
    movieTitle: `Seven Years in Tibet`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/Seven_Years_in_Tibet.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/seven-years-in-tibet.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/Seven_Years_in_Tibet.jpg`,
    backgroundColor: `#C6CADF`,
    description: `True story of Heinrich Harrer, an Austrian mountain climber who became friends with the Dalai Lama at the time of China's takeover of Tibet.`,
    rating: 3.6,
    scoresCount: 112612,
    director: `Jean-Jacques Annaud`,
    starring: [
      `Brad Pitt`,
      `David Thewlis`,
      `BD Wong`
    ],
    runtime: 136,
    genre: `Adventure`,
    released: 1997,
    id: 1,
    isFavorite: false,
    videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
    previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
  },
  {
    movieTitle: `What We Do in the Shadows`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/What-We-Do-in-the-Shadows.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/what-we-do-in-the-shadows.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/What-We-Do-in-the-Shadows.jpg`,
    backgroundColor: `#A39E81`,
    description: `A look into the daily (or rather, nightly) lives of three vampires who've lived together for over 100 years, in Staten Island.`,
    rating: 4.2,
    scoresCount: 6173,
    director: `Jemaine Clement`,
    starring: [
      `Kayvan Novak`,
      `Matt Berry`,
      `Natasia Demetriou`
    ],
    runtime: 30,
    genre: `Comedy`,
    released: 2019,
    id: 2,
    isFavorite: false,
    videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
    previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    movieTitle: `Johnny English`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/Johnny_English.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/johnny-english.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/Johnny_English.jpg`,
    backgroundColor: `#F0DBA2`,
    description: `After a sudden attack on the MI5, Johnny English, Britain's most confident yet unintelligent spy, becomes Britain's only spy.`,
    rating: 3.1,
    scoresCount: 136843,
    director: `Peter Howitt`,
    starring: [
      `Rowan Atkinson`,
      `John Malkovich`,
      `Natalie Imbruglia`
    ],
    runtime: 88,
    genre: `Comedy`,
    released: 2003,
    id: 3,
    isFavorite: false,
    videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
    previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    movieTitle: `Bronson`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/bronson.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/bronson.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/bronson.jpg`,
    backgroundColor: `#73B39A`,
    description: `A young man who was sentenced to seven years in prison for robbing a post office ends up spending three decades in solitary confinement. During this time, his own personality is supplanted by his alter-ego, Charles Bronson.`,
    rating: 3.6,
    scoresCount: 109661,
    director: `Nicolas Winding Refn`,
    starring: [
      `Tom Hardy`,
      `Kelly Adams`,
      `Luing Andrews`
    ],
    runtime: 92,
    genre: `Action`,
    released: 2008,
    id: 4,
    isFavorite: false,
    videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
    previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
  },
  {
    movieTitle: `Macbeth`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/Macbeth.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/macbeth.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/Macbeth.jpg`,
    backgroundColor: `#F1E9CE`,
    description: `Macbeth, the Thane of Glamis, receives a prophecy from a trio of witches that one day he will become King of Scotland. Consumed by ambition and spurred to action by his wife, Macbeth murders his king and takes the throne for himself.`,
    rating: 3.3,
    scoresCount: 48798,
    director: `Justin Kurzel`,
    starring: [
      `Michael Fassbender`,
      `Marion Cotillard`,
      `Jack Madigan`
    ],
    runtime: 113,
    genre: `Drama`,
    released: 2015,
    id: 5,
    isFavorite: false,
    videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
    previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
  },
  {
    movieTitle: `Seven Years in Tibet`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/Seven_Years_in_Tibet.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/seven-years-in-tibet.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/Seven_Years_in_Tibet.jpg`,
    backgroundColor: `#C6CADF`,
    description: `True story of Heinrich Harrer, an Austrian mountain climber who became friends with the Dalai Lama at the time of China's takeover of Tibet.`,
    rating: 3.6,
    scoresCount: 112612,
    director: `Jean-Jacques Annaud`,
    starring: [
      `Brad Pitt`,
      `David Thewlis`,
      `BD Wong`
    ],
    runtime: 136,
    genre: `Adventure`,
    released: 1997,
    id: 6,
    isFavorite: false,
    videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
    previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
  },
  {
    movieTitle: `What We Do in the Shadows`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/What-We-Do-in-the-Shadows.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/what-we-do-in-the-shadows.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/What-We-Do-in-the-Shadows.jpg`,
    backgroundColor: `#A39E81`,
    description: `A look into the daily (or rather, nightly) lives of three vampires who've lived together for over 100 years, in Staten Island.`,
    rating: 4.2,
    scoresCount: 6173,
    director: `Jemaine Clement`,
    starring: [
      `Kayvan Novak`,
      `Matt Berry`,
      `Natasia Demetriou`
    ],
    runtime: 30,
    genre: `Comedy`,
    released: 2019,
    id: 7,
    isFavorite: false,
    videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
    previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    movieTitle: `Johnny English`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/Johnny_English.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/johnny-english.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/Johnny_English.jpg`,
    backgroundColor: `#F0DBA2`,
    description: `After a sudden attack on the MI5, Johnny English, Britain's most confident yet unintelligent spy, becomes Britain's only spy.`,
    rating: 3.1,
    scoresCount: 136843,
    director: `Peter Howitt`,
    starring: [
      `Rowan Atkinson`,
      `John Malkovich`,
      `Natalie Imbruglia`
    ],
    runtime: 88,
    genre: `Comedy`,
    released: 2003,
    id: 8,
    isFavorite: false,
    videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
    previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    movieTitle: `Bronson`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/bronson.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/bronson.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/bronson.jpg`,
    backgroundColor: `#73B39A`,
    description: `A young man who was sentenced to seven years in prison for robbing a post office ends up spending three decades in solitary confinement. During this time, his own personality is supplanted by his alter-ego, Charles Bronson.`,
    rating: 3.6,
    scoresCount: 109661,
    director: `Nicolas Winding Refn`,
    starring: [
      `Tom Hardy`,
      `Kelly Adams`,
      `Luing Andrews`
    ],
    runtime: 92,
    genre: `Action`,
    released: 2008,
    id: 9,
    isFavorite: false,
    videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
    previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
  },
  {
    movieTitle: `Macbeth`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/Macbeth.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/macbeth.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/Macbeth.jpg`,
    backgroundColor: `#F1E9CE`,
    description: `Macbeth, the Thane of Glamis, receives a prophecy from a trio of witches that one day he will become King of Scotland. Consumed by ambition and spurred to action by his wife, Macbeth murders his king and takes the throne for himself.`,
    rating: 3.3,
    scoresCount: 48798,
    director: `Justin Kurzel`,
    starring: [
      `Michael Fassbender`,
      `Marion Cotillard`,
      `Jack Madigan`
    ],
    runtime: 113,
    genre: `Drama`,
    released: 2015,
    id: 10,
    isFavorite: false,
    videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
    previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
  },
  {
    movieTitle: `Seven Years in Tibet`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/Seven_Years_in_Tibet.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/seven-years-in-tibet.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/Seven_Years_in_Tibet.jpg`,
    backgroundColor: `#C6CADF`,
    description: `True story of Heinrich Harrer, an Austrian mountain climber who became friends with the Dalai Lama at the time of China's takeover of Tibet.`,
    rating: 3.6,
    scoresCount: 112612,
    director: `Jean-Jacques Annaud`,
    starring: [
      `Brad Pitt`,
      `David Thewlis`,
      `BD Wong`
    ],
    runtime: 136,
    genre: `Adventure`,
    released: 1997,
    id: 11,
    isFavorite: false,
    videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
    previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
  },
  {
    movieTitle: `What We Do in the Shadows`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/What-We-Do-in-the-Shadows.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/what-we-do-in-the-shadows.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/What-We-Do-in-the-Shadows.jpg`,
    backgroundColor: `#A39E81`,
    description: `A look into the daily (or rather, nightly) lives of three vampires who've lived together for over 100 years, in Staten Island.`,
    rating: 4.2,
    scoresCount: 6173,
    director: `Jemaine Clement`,
    starring: [
      `Kayvan Novak`,
      `Matt Berry`,
      `Natasia Demetriou`
    ],
    runtime: 30,
    genre: `Comedy`,
    released: 2019,
    id: 12,
    isFavorite: false,
    videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
    previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    movieTitle: `Johnny English`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/Johnny_English.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/johnny-english.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/Johnny_English.jpg`,
    backgroundColor: `#F0DBA2`,
    description: `After a sudden attack on the MI5, Johnny English, Britain's most confident yet unintelligent spy, becomes Britain's only spy.`,
    rating: 3.1,
    scoresCount: 136843,
    director: `Peter Howitt`,
    starring: [
      `Rowan Atkinson`,
      `John Malkovich`,
      `Natalie Imbruglia`
    ],
    runtime: 88,
    genre: `Comedy`,
    released: 2003,
    id: 13,
    isFavorite: false,
    videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
    previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    movieTitle: `Bronson`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/bronson.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/bronson.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/bronson.jpg`,
    backgroundColor: `#73B39A`,
    description: `A young man who was sentenced to seven years in prison for robbing a post office ends up spending three decades in solitary confinement. During this time, his own personality is supplanted by his alter-ego, Charles Bronson.`,
    rating: 3.6,
    scoresCount: 109661,
    director: `Nicolas Winding Refn`,
    starring: [
      `Tom Hardy`,
      `Kelly Adams`,
      `Luing Andrews`
    ],
    runtime: 92,
    genre: `Action`,
    released: 2008,
    id: 14,
    isFavorite: false,
    videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
    previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
  },
  {
    movieTitle: `Macbeth`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/Macbeth.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/macbeth.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/Macbeth.jpg`,
    backgroundColor: `#F1E9CE`,
    description: `Macbeth, the Thane of Glamis, receives a prophecy from a trio of witches that one day he will become King of Scotland. Consumed by ambition and spurred to action by his wife, Macbeth murders his king and takes the throne for himself.`,
    rating: 3.3,
    scoresCount: 48798,
    director: `Justin Kurzel`,
    starring: [
      `Michael Fassbender`,
      `Marion Cotillard`,
      `Jack Madigan`
    ],
    runtime: 113,
    genre: `Drama`,
    released: 2015,
    id: 15,
    isFavorite: false,
    videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
    previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
  },
  {
    movieTitle: `Seven Years in Tibet`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/Seven_Years_in_Tibet.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/seven-years-in-tibet.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/Seven_Years_in_Tibet.jpg`,
    backgroundColor: `#C6CADF`,
    description: `True story of Heinrich Harrer, an Austrian mountain climber who became friends with the Dalai Lama at the time of China's takeover of Tibet.`,
    rating: 3.6,
    scoresCount: 112612,
    director: `Jean-Jacques Annaud`,
    starring: [
      `Brad Pitt`,
      `David Thewlis`,
      `BD Wong`
    ],
    runtime: 136,
    genre: `Adventure`,
    released: 1997,
    id: 16,
    isFavorite: false,
    videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
    previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
  },
  {
    movieTitle: `What We Do in the Shadows`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/What-We-Do-in-the-Shadows.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/what-we-do-in-the-shadows.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/What-We-Do-in-the-Shadows.jpg`,
    backgroundColor: `#A39E81`,
    description: `A look into the daily (or rather, nightly) lives of three vampires who've lived together for over 100 years, in Staten Island.`,
    rating: 4.2,
    scoresCount: 6173,
    director: `Jemaine Clement`,
    starring: [
      `Kayvan Novak`,
      `Matt Berry`,
      `Natasia Demetriou`
    ],
    runtime: 30,
    genre: `Comedy`,
    released: 2019,
    id: 17,
    isFavorite: false,
    videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
    previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    movieTitle: `Johnny English`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/Johnny_English.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/johnny-english.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/Johnny_English.jpg`,
    backgroundColor: `#F0DBA2`,
    description: `After a sudden attack on the MI5, Johnny English, Britain's most confident yet unintelligent spy, becomes Britain's only spy.`,
    rating: 3.1,
    scoresCount: 136843,
    director: `Peter Howitt`,
    starring: [
      `Rowan Atkinson`,
      `John Malkovich`,
      `Natalie Imbruglia`
    ],
    runtime: 88,
    genre: `Comedy`,
    released: 2003,
    id: 18,
    isFavorite: false,
    videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
    previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    movieTitle: `Bronson`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/bronson.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/bronson.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/bronson.jpg`,
    backgroundColor: `#73B39A`,
    description: `A young man who was sentenced to seven years in prison for robbing a post office ends up spending three decades in solitary confinement. During this time, his own personality is supplanted by his alter-ego, Charles Bronson.`,
    rating: 3.6,
    scoresCount: 109661,
    director: `Nicolas Winding Refn`,
    starring: [
      `Tom Hardy`,
      `Kelly Adams`,
      `Luing Andrews`
    ],
    runtime: 92,
    genre: `Action`,
    released: 2008,
    id: 19,
    isFavorite: false,
    videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
    previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
  },
  {
    movieTitle: `Macbeth`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/Macbeth.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/macbeth.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/Macbeth.jpg`,
    backgroundColor: `#F1E9CE`,
    description: `Macbeth, the Thane of Glamis, receives a prophecy from a trio of witches that one day he will become King of Scotland. Consumed by ambition and spurred to action by his wife, Macbeth murders his king and takes the throne for himself.`,
    rating: 3.3,
    scoresCount: 48798,
    director: `Justin Kurzel`,
    starring: [
      `Michael Fassbender`,
      `Marion Cotillard`,
      `Jack Madigan`
    ],
    runtime: 113,
    genre: `Drama`,
    released: 2015,
    id: 20,
    isFavorite: false,
    videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
    previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
  },
  {
    movieTitle: `Seven Years in Tibet`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/Seven_Years_in_Tibet.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/seven-years-in-tibet.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/Seven_Years_in_Tibet.jpg`,
    backgroundColor: `#C6CADF`,
    description: `True story of Heinrich Harrer, an Austrian mountain climber who became friends with the Dalai Lama at the time of China's takeover of Tibet.`,
    rating: 3.6,
    scoresCount: 112612,
    director: `Jean-Jacques Annaud`,
    starring: [
      `Brad Pitt`,
      `David Thewlis`,
      `BD Wong`
    ],
    runtime: 136,
    genre: `Adventure`,
    released: 1997,
    id: 21,
    isFavorite: false,
    videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
    previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
  },
  {
    movieTitle: `What We Do in the Shadows`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/What-We-Do-in-the-Shadows.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/what-we-do-in-the-shadows.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/What-We-Do-in-the-Shadows.jpg`,
    backgroundColor: `#A39E81`,
    description: `A look into the daily (or rather, nightly) lives of three vampires who've lived together for over 100 years, in Staten Island.`,
    rating: 4.2,
    scoresCount: 6173,
    director: `Jemaine Clement`,
    starring: [
      `Kayvan Novak`,
      `Matt Berry`,
      `Natasia Demetriou`
    ],
    runtime: 30,
    genre: `Comedy`,
    released: 2019,
    id: 22,
    isFavorite: false,
    videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
    previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    movieTitle: `Johnny English`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/Johnny_English.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/johnny-english.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/Johnny_English.jpg`,
    backgroundColor: `#F0DBA2`,
    description: `After a sudden attack on the MI5, Johnny English, Britain's most confident yet unintelligent spy, becomes Britain's only spy.`,
    rating: 3.1,
    scoresCount: 136843,
    director: `Peter Howitt`,
    starring: [
      `Rowan Atkinson`,
      `John Malkovich`,
      `Natalie Imbruglia`
    ],
    runtime: 88,
    genre: `Comedy`,
    released: 2003,
    id: 23,
    isFavorite: false,
    videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
    previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    movieTitle: `Bronson`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/bronson.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/bronson.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/bronson.jpg`,
    backgroundColor: `#73B39A`,
    description: `A young man who was sentenced to seven years in prison for robbing a post office ends up spending three decades in solitary confinement. During this time, his own personality is supplanted by his alter-ego, Charles Bronson.`,
    rating: 3.6,
    scoresCount: 109661,
    director: `Nicolas Winding Refn`,
    starring: [
      `Tom Hardy`,
      `Kelly Adams`,
      `Luing Andrews`
    ],
    runtime: 92,
    genre: `Action`,
    released: 2008,
    id: 24,
    isFavorite: false,
    videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
    previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
  },
  {
    movieTitle: `Macbeth`,
    movieImg: `https://es31-server.appspot.com/wtw/static/film/poster/Macbeth.jpg`,
    previewImage: `https://es31-server.appspot.com/wtw/static/film/preview/macbeth.jpg`,
    backgroundImage: `https://es31-server.appspot.com/wtw/static/film/background/Macbeth.jpg`,
    backgroundColor: `#F1E9CE`,
    description: `Macbeth, the Thane of Glamis, receives a prophecy from a trio of witches that one day he will become King of Scotland. Consumed by ambition and spurred to action by his wife, Macbeth murders his king and takes the throne for himself.`,
    rating: 3.3,
    scoresCount: 48798,
    director: `Justin Kurzel`,
    starring: [
      `Michael Fassbender`,
      `Marion Cotillard`,
      `Jack Madigan`
    ],
    runtime: 113,
    genre: `Drama`,
    released: 2015,
    id: 25,
    isFavorite: false,
    videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
    previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
  }


];

export default films;
