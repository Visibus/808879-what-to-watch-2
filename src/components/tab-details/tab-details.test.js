import TabDetails from "./tab-details";
import films from "../../mocks/films";

it(`TabDetails component renders correctly`, () => {
  const tree = window.renderer
    .create(
        <TabDetails currentTab={`Details`} indexTab={`Details`} film={films[0]}>
        </TabDetails>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
