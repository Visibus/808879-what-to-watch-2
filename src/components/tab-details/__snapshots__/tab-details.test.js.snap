// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabDetails component renders correctly 1`] = `
<div
  className="movie-card__text movie-card__row"
>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Director
      </strong>
      <span
        className="movie-card__details-value"
      >
        Jean-Jacques Annaud
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Starring
      </strong>
      <span
        className="movie-card__details-value"
      >
        Brad Pitt,
David Thewlis,
BD Wong
      </span>
    </p>
  </div>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Run Time
      </strong>
      <span
        className="movie-card__details-value"
      >
        2h 16m
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Genre
      </strong>
      <span
        className="movie-card__details-value"
      >
        Adventure
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Released
      </strong>
      <span
        className="movie-card__details-value"
      >
        1997
      </span>
    </p>
  </div>
</div>
`;
